### DISCUSSION TITLE
Registry - When to change the shardId and shardLabel

### DISCUSSION THREAD

**Discussion by maheshkumargangula** [Sunbird-RC/community#128] [Issues]
@dileepbapat - We have `sharedId` and `shardLabel` in the application.yml file with default values. How to use these configuration properties.

When we should define a new value? If there are multiple containers running?

Cc: @vrayulu

**Comment by dileepbapat** [comment]
Not for multiple containers, multiple containers can point to same db. it is if you configure shard manager based on user keys to store in different databases.
